pkg/controller/directvolumemigration/rsync.go

```diff
@@ -948,6 +955,9 @@ func (t *Task) createRsyncClientPods() error {
 			})
 			rsyncCommand := []string{"rsync"}
 			rsyncCommand = append(rsyncCommand, t.getRsyncOptions()...)
+			if vol.verify {
+				rsyncCommand = append(rsyncCommand, "--checksum")
```

Comment:
I have a question here: if a pod already exists without the checksum command on the source side, https://github.com/konveyor/mig-controller/pull/890/files#diff-7299a3c4e600e2733571d1d39c64bbfe6c7e448a6bb15aa472514a173d3095c0R1015 we would not be creating one. So we are promissing user to run with checksum but dont actual run it be checksum in that case?

Reply:
this will work for 90% of the case I think but might need a followup to fix it for good